packages/energy-api-influxdb/src/reads/base-reads.controller.ts

```diff
@@ -47,6 +52,15 @@ export abstract class BaseReadsController {
     return res;
   }
 
+  @Get('/:meter/last')
```

Comment:
IMO `last` is not the correct keyword, since it implies that nothing else will follow.
Please rename it to `latest`, which means "last to date, more could follow".

Reply:
Makes sense